x-pack/plugins/maps/public/classes/layers/blended_vector_layer/blended_vector_layer.ts

```diff
@@ -305,6 +300,7 @@ export class BlendedVectorLayer extends VectorLayer implements IVectorLayer {
       prevDataRequest: this.getDataRequest(dataRequestId),
       nextMeta: searchFilters,
       extentAware: source.isFilterByMapBounds(),
+      getUpdateDueToTimeslice: this._getUpdateDueToTimesliceFromSourceRequestMeta,
```

Comment:
Can you wrap these calls in a lambda instead? Passing in methods is hard to follow why it works (it's not a "real" method on the prototype, but an instance variable. Ah, that's why it works). Or at least make it consistent, and make it instance variables everywhere. (e.g. `_getUpdateDueToTimesliceFromTileMeta` is still a method, not a function. just works because it has a constant return).